spread-tests/install-go.sh

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env bash
+
+if [[ -v ENSUREGO ]]; then
+    echo "Go already installed"
+else
+    echo "Installing go"
+    snap wait system seed.loaded
+    snap install go --classic
```

Comment:
Check how snapd does this

Reply:
Should be a similar approach but regardless we don't really need to go a different route here. We will likely restructure this part to test with varius versions in the future.

Reply:
Michael Vogt confirmed this is basically the right way to install the snap version of go for use in spread tests. I should probably install from apt and not snap however.